architecture/topics/router_environment_variables.adoc

```diff
@@ -75,6 +75,12 @@ endif::[]
 |`ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` |  | Set to `true` to relax the namespace ownership policy.
 |`ROUTER_STRICT_SNI` |  | xref:strict-sni[strict-sni]
 |`ROUTER_CIPHERS` | intermediate  | Specify the set of xref:ciphers[ciphers] supported by bind.
+|`ROUTER_HAPROXY_CONFIG_MANAGER` | | When set to `true` or `TRUE`, enables a dynamic config manager with haproxy. This allows certain types of routes to be managed dynamically by the config manager and reduces the amount of haproxy router reloads.
```

Comment:
I'd s/amount/number

Reply:
Is this value "False" by default?

Reply:
Catching up on emails - yes this is false by default.